src/pages/tug/index.page.js

```diff
@@ -0,0 +1,218 @@
+      tugOrganizationChartUrl,
+      howToBecomeTitle,
+      howToBecome,
+      progressTitle,
+      progressSummary,
+      progressStep1,
+      progressStep2,
+      peopleMemberTitle,
+      peopleMemberSummary,
+      peopleLeaderTitle,
+      peopleLeaderSummary,
+      tmc,
+      leader,
+    },
+  };
+};
+
+const People = ({
+  title,
+  description,
+  applyUrl,
+  tugOrganizationChartTitle,
+  tugOrganizationChartUrl,
+  howToBecomeTitle,
+  howToBecome,
+  progressTitle,
+  progressSummary,
+  progressStep1,
+  progressStep2,
+  peopleMemberTitle,
+  peopleMemberSummary,
+  peopleLeaderTitle,
+  peopleLeaderSummary,
+  tmc,
+  leader,
+}) => {
+  const howToBecomeDom = [];
+
+  howToBecome.forEach((item, index) => {
+    howToBecomeDom.push(<HowToBecome key={uuidV4()} {...item} />);
+    if (index !== howToBecome.length - 1) {
+      howToBecomeDom.push(
+        <div key={uuidV4()} className={styles.how_to_become_list_arrow}>
+          <img src="/images/people/arrow.svg" alt=">" />
+        </div>
+      );
+    }
+  });
+
+  return (
+    <>
+      <TugHead title={title} description={description} />
+
+      <CoreLayout>
+        <div className={styles.header}>
+          <Container>
+            <div className={styles.header_content}>
+              <div className={styles.header_content_left}>
+                <div className={styles.header_content_left_title}>
+                  <img src="/images/people/tug-logo.svg" alt="" />
+                </div>
+                <div className={styles.header_content_left_summary}>{description}</div>
+                <div className={styles.header_content_left_summary}>
+                  <Button as={MyLink} href={applyUrl}>
+                    加入 TUG
+                  </Button>
+                </div>
+              </div>
+              <div className={styles.header_content_right}>
+                <img src="/images/people/header-right.svg" alt="" />
+              </div>
+            </div>
+          </Container>
+        </div>
+
+        <Container className={styles.organization_chart}>
+          <div className={styles.organization_chart_content}>
+            <div className={styles.organization_chart_content_title}>{tugOrganizationChartTitle}</div>
+            <div className={styles.organization_chart_content_image}>
+              <MyLink href={tugOrganizationChartUrl}>
+                <img src={tugOrganizationChartUrl} alt={tugOrganizationChartTitle} />
+              </MyLink>
+            </div>
+          </div>
+        </Container>
+
+        <div className={styles.how_to_become_wrapper}>
+          <Container className={styles.how_to_become}>
+            <div className={styles.how_to_become_title}>{howToBecomeTitle}</div>
+            <div className={styles.how_to_become_list}>{howToBecomeDom}</div>
+          </Container>
+        </div>
+
+        <Container className={styles.progress}>
+          <div className={styles.progress_left}>
```

Comment:
I think it's better to split codes into layout component and detail components like
```jsx
<>
  <Organization />
  <HowToBecome />
  <Progress />
  ...
</>
```